pkg/components/server.go

```diff
@@ -143,7 +150,24 @@ func newServerConfigured(
 					Fmt: ytconfig.ConfigFormatYson,
 				},
 			}),
+
+		componentContainerPorts: []corev1.ContainerPort{
+			{
+				Name:          consts.YTMonitoringContainerPortName,
```

Comment:
Suppose we should mention this backward incompatibility rename in release notes after merge.